### ISSUE TITLE
I don't find script_name.py file

### ISSUE THREAD

**Issue by andeyeluguo** [kai-xie/Neural-Network-Model-Compressing#10]
you say that 'python script_name.py' ,But I don't find any file name script_name.py。Thank you、

**Comment by kai-xie** [comment]
@andeyeluguo you are right that there is no `script_name.py`, it just stands for a random python scripts in the `compression_scripts` folder. Will fix my word later.
